src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncSpillTests.cs

```diff
@@ -554,7 +554,6 @@ .maxstack  2
         public void SpillNestedUnary()
         {
             var source = @"
-#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
```

Comment:
You've removed `#pragma warning disable CS1998` from tests which I think is nice, but looks like there are still many `#pragma warning disable 1998` (without the `CS` prefix) which could be cleaned up as well.